src/views/Error.vue

```diff
@@ -12,25 +12,27 @@
     <!-- Browser error starts -->
     <div v-if="isBrowserError">
 
-      <!-- TODO
-      remove hardcoded link -> either parameterize it, or enter a link to youtube -->
-      <div class='embed-container'>
-        <iframe src='https://www.youtube.com/embed/FLOwzot27XM' frameborder='0' allowfullscreen>
-        </iframe>
-      </div>
-
       <!-- failsafe begins -->
       <div v-if="hasFailSafe">
 
+        <!-- TODO - Adding loading spinner till YT video loads -->
         <!-- failsafe G-form begins -->
-        <div v-if="isFailSafeGform">
+        <div v-if="isFailSafeGform && isVideoIdFetched" >
```

Comment:
would be better to decouple video from failsafe
and rename the variable from fetch to videoIdpresent or something

Reply:
- Okay so that means even if there's no fail-safe g-form, the student can atleast watch the normal youtube video?
- will rename